lib/peep.rb

```diff
@@ -0,0 +1,75 @@
+require 'pg'
+require './lib/database'
+
+class Peep
+
+    attr_reader :id, :message, :name, :created_at
+
+    def initialize(id:, message:, name:, created_at:)
+        @id  = id
+        @message = message
+        @name = name
+        @created_at = created_at
+    end
+
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
```

Comment:
This if else statement is repeated in every function, could this be pulled out into its own function?